adhocracy4/api/permissions.py

```diff
@@ -19,3 +22,85 @@ def has_object_permission(self, request, view, obj):
             obj.creator == request.user
             or request.user.is_superuser
             or obj.project.has_moderator(request.user))
+
+
+RulesMethodMap = namedtuple('RulesMethodMap',
+                            ['GET', 'OPTIONS', 'HEAD', 'POST',
+                             'PUT', 'PATCH', 'DELETE'])
+
+
+class RulesPermission(permissions.BasePermission):
+    """
+    Combining drf permissions with django-rules.
+    """
+
+    default_rules_method_map = RulesMethodMap(
+        GET='{app_label}.view_{model_name}',
+        OPTIONS='{app_label}.view_{model_name}',
+        HEAD='{app_label}.view_{model_name}',
+        POST='{app_label}.create_{model_name}',
+        PUT='{app_label}.change_{model_name}',
+        PATCH='{app_label}.change_{model_name}',
+        DELETE='{app_label}.delete_{model_name}',
```

Comment:
You mean as verbs in those permissions. I took those from [DRF DjangoModelPermission class](https://github.com/encode/django-rest-framework/blob/master/rest_framework/permissions.py#L94), which probably choose those names because the are the [default permission names](https://docs.djangoproject.com/en/1.11/ref/models/options/#default-permissions).

That was more informal than a counter argument. I like the CRUD wording as well, but the other one is well established in the framework. In any case, we should rework our existing permissions to match the naming shema that we choose here :weight_lifting_woman:.